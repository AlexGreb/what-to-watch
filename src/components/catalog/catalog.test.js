import React from 'react';
import renderer from 'react-test-renderer';
import Catalog from './catalog.jsx';

describe(`Catalog`, () => {
  it(`Should render correctly a catalog`, () => {
    const tree = renderer.create(<Catalog/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

});

