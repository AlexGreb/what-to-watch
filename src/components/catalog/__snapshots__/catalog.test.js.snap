// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Catalog Should render correctly a catalog 1`] = `
<section
  className="catalog"
>
  <h2
    className="catalog__title visually-hidden"
  >
    Catalog
  </h2>
  <ul
    className="catalog__genres-list"
  >
    <li
      className="catalog__genres-item catalog__genres-item--active"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        All genres
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Comedies
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Crime
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Documentary
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Dramas
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Horror
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Kids & Family
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Romance
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Sci-Fi
      </a>
    </li>
    <li
      className="catalog__genres-item"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Thrillers
      </a>
    </li>
  </ul>
  <div
    className="catalog__movies-list"
  >
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/pulp-fiction.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Pulp Fiction
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/matrix.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Matrix
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/revenant.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          The Revenant
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/legend.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Legend
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/bohemian_rhapsody.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Bohemian Rhapsody
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/no-country-for-old-men.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          No Country for Old Men
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/snatch.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Snatch
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/johnny-english.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Johnny English
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/bronson.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Bronson
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Seven Years in Tibet
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/aviator.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Aviator
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/A_Star_Is_Born.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          A Star Is Born
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/orlando.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Orlando
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/beach.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Beach
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/we-need-to-talk-about-kevin.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          We need to talk about Kevin
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/gangs_of_new_york.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Gangs of new york
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/fantastic-beasts-the-crimes-of-grindelwald.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Fantastic Beasts: The Crimes of Grindelwald
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/macbeth.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Macbeth
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          What We Do in the Shadows
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/shutter-island.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Shutter Island
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/war-of-the-worlds.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          War of the Worlds
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/dardjeeling_limited.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Dardjeeling Limited
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/Once_Upon_a_Time_in_America.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Once Upon a Time in America
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/midnight-special.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Midnight Special
        </a>
      </h3>
    </article>
    <article
      className="small-movie-card catalog__movies-card"
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src="https://es31-server.appspot.com/wtw/static/film/preview/moonrise-kingdom.jpg"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <a
          className="small-movie-card__link"
          href="movie-page.html"
        >
          Moonrise Kingdom
        </a>
      </h3>
    </article>
  </div>
  <div
    className="catalog__more"
  >
    <button
      className="catalog__button"
      type="button"
    >
      Show more
    </button>
  </div>
</section>
`;
